examples/lang/erc20/src/lib.rs

```diff
@@ -34,18 +34,16 @@ use ink_lang::contract;
 /// Events deposited by the ERC20 token contract.
 #[derive(Encode, Decode)]
 enum Event {
```

Comment:
You can use the new event syntax. The remaining issue with it has been fixed.

Reply:
Done :) (but not so pretty)